Plugin/Framework/Search/Request/AbstractHandler.php

```diff
@@ -281,13 +280,6 @@ private function getCmpProductIds(FacetInterface $facet, $pageNum, $limit)
                 $limit
             );
             return $res ? CategoryMerchandising::parseProductIds($res) : null;
-        } catch (MissingCookieException $e) {
```

Comment:
Why is this removed?

Reply:
This was never thrown in the first place